content/en/administrators_guide/build.md

```diff
@@ -9,15 +9,15 @@ further_reading:
 
 After you plan your Datadog installation design and best practices, concentrate on the construction of Datadog itself, understanding what needs to be installed, and the best way to achieve that.
 
-As you scale your IT footprint, it's important to develop standards, methods, and practices for how software is installed and used. Engineering effort is dedicated to developing precise steps for installing and configuring software in a repeatable, standardized manner, while still allowing for a significant level of flexibility. Without these standards, reliability can be impacted. In this section, we describe how Datadog can most efficiently interact with those standards, methods, and practices.
+As your IT footprint grows, establish standards for software installation and usage. Develop precise, repeatable steps for configuring software while maintaining flexibility. This ensures reliability. This section explains how Datadog can efficiently integrate with these standards.
```

Comment:
I'm loving the improved conciseness here, but I was having some trouble following the relationships between these sentences. I'm wondering if we can reinstate some of the transitional language between sentences, maybe like:

```suggestion
As your IT footprint grows, you need to establish standards for software installation and usage. To do this, it's important to develop precise, repeatable steps for reliably configuring software while maintaining the flexibility you need. This section explains how Datadog can efficiently integrate with these standards.
```

Reply:
that is much better thank you!